src/universal/common/components/FormattedLink/FormattedLink.tsx

```diff
@@ -1,14 +1,21 @@
 import * as React from 'react';
 
 interface IFormattedLinkProps {
-  id: string;
-  children: any;
+  id?: string;
+  href?: string;
+  children?: any;
 }
 
 export const FormattedLink = (props: IFormattedLinkProps, context) => {
-  const {intl: {formatMessage}} = context;
-  const {id, children} = props;
-  const formattedHref = formatMessage({id});
+  const {intl: {formatMessage, locale}} = context;
+  const {id, href, children} = props;
+  let formattedHref = '';
+  if (id) {
+    formattedHref = formatMessage({id});
+  } else if (href) {
+    const urlParts = [locale, ...href.split('/')].filter(part => !!part);
```

Comment:
If `href` is present, will form a new url appended with the locale and non-empty parts of the url.